packages/tasit-act/src/ethers.test.js

```diff
@@ -0,0 +1,92 @@
+import { expect, assert } from "chai";
+import { ethers } from "ethers";
+// Note: This file is originally genarated by `tasit-contracts` and was pasted here manually
+// See https://github.com/tasitlabs/TasitSDK/issues/45
+import { abi as contractABI } from "./helpers/SimpleStorage.json";
+// Note: Under the current `tasit-contracts` setup SimpleStorage aways will deployed with this address
+// See https://github.com/tasitlabs/TasitSDK/pull/59#discussion_r242258739
+const contractAddress = "0x6C4A015797DDDd87866451914eCe1e8b19261931";
+
+let provider, wallet, contract;
+
+describe("ethers.js", () => {
+  beforeEach(
+    "instantiate provider, wallet and contract objects",
+    async function() {
```

Comment:
Is it guaranteed that beforeEach waits for the async function here to complete?

Reply:
Yes. https://mochajs.org/#using-async--await

Reply:
Cool - thanks!